lib/cash-out/cash-out-helper.js

```diff
@@ -11,14 +11,17 @@ module.exports = { redeemableTxs, toObj, toDb, REDEEMABLE_AGE }
 const mapValuesWithKey = _.mapValues.convert({cap: false})
 
 function convertBigNumFields (obj) {
-  const convert = (value, key) => _.includes(key, [
-    'cryptoAtoms',
-    'fiat',
-    'commissionPercentage',
-    'rawTickerPrice'
-  ])
-    ? value.toString()
-    : value
+  const convert = (value, key) => {
+    if (_.includes(key, [ 'cryptoAtoms', 'receivedCryptoAtoms', 'fiat' ])) {
+      return value.toString()
+    }
+
+    if (_.includes(key, [ 'commissionPercentage', 'rawTickerPrice' ]) && value) {
```

Comment:
Why not always check for existence of value when returning ``toString()``? Also, check for ``isNil``, not raw value since it could be 0.

Reply:
Since the other fields should never be null I would rather them to break, added comment on it.